src/components/VideoHero/VideoHero.tsx

```diff
@@ -45,7 +43,7 @@ export const VideoHero: React.FC = () => {
 
   const handlePlaybackDataLoaded = () => {
     setTimeout(() => {
-      setDisplayControls(true)
+      setDisabledControls(false)
```

Comment:
After testing it out I've noticed that after the most recent change to the video hero this doesn't make much sense. Previously this button was controlling the playback inside the video hero but now it only navigates you to the video. So I think we could actually make the button enabled always (i.e. as long as the `coverVideo` is loaded). Let's remove the logic for disabling this button